src/test/java/uk/gov/hmcts/reform/unspec/handler/callback/RespondToClaimCallbackHandlerTest.java

```diff
@@ -0,0 +1,122 @@
+package uk.gov.hmcts.reform.unspec.handler.callback;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.autoconfigure.validation.ValidationAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.callback.CallbackType;
+import uk.gov.hmcts.reform.unspec.validation.DateOfBirthValidator;
+
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.lang.String.format;
+import static java.time.LocalDate.now;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.unspec.handler.callback.RespondToClaimCallbackHandler.CLAIMANT_RESPONSE_DEADLINE;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {
+    RespondToClaimCallbackHandler.class,
+    JacksonAutoConfiguration.class,
+    ValidationAutoConfiguration.class,
+    DateOfBirthValidator.class
+})
+class RespondToClaimCallbackHandlerTest extends BaseCallbackHandlerTest {
+
+    @Autowired
+    private RespondToClaimCallbackHandler handler;
+
+    @Nested
+    class MidEventCallback {
+
+        @ParameterizedTest
+        @ValueSource(strings = {"individualDateOfBirth", "soleTraderDateOfBirth"})
+        void shouldReturnError_whenDateOfBirthIsInTheFuture(String dateOfBirthField) {
```

Comment:
do I care about this depth of testing now that the validation exists elsewhere in a common service? The future / past / current stuff should be tested in that new service. Here I am not sure it requires tetsing again. Thoughts?

Reply:
I am open to any option. can leave one success and one failure scenario there.